### PR TITLE
Implement batch revive

### PR DESCRIPTION
**Pull request by democrazyfi**

### CODE REVIEW

**Comment by Ledespana** [comment]
Screenshot 2024-04-24 at 8 17 14 PM
The revive button doesn't seem to do anything (on individual ones)

File: frontend/components/LSP8Panel.tsx [hunk 1]
```diff
@@ -109,7 +113,13 @@ const LSP8Panel: React.FC<LSP8PanelProps> = ({
         .connect(signer)
         .execute(0, tokenAddress, 0, lsp8Tx, { gasLimit: 400_00 });
 
-      onReviveSuccess(tokenAddress, tokenData.tokenId as string);
+      setInProcessingText('Marking unsafe...');
+      await LSP1GraveForwarderContract.removeTokenFromAllowlist(tokenAddress, {
+        gasLimit: 400_00,
+      });
```
Comment (Ledespana) [inline frontend/components/LSP8Panel.tsx:119]: why are we removing from allow list before reviving? shouldn't it we adding it?

**Comment by Ledespana** [comment]
Max size issue 
Screenshot 2024-04-26 at 9 57 08 AM

**Comment by Ledespana** [comment]
I think there are a few bugs on the flow:
1- On individual revivals, after it is revived, there is a third transaction for removing from allow list
2- every individual revive is 3 transactions.
3- After reviving 2, the modal closes even though there are 3
4- sizing issue

**Review by tehnalogos — APPROVED** [review]

**Pull request merged**